ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/claimantresponse/ResponseAcceptationAssert.java

```diff
@@ -16,7 +16,10 @@ public ResponseAcceptationAssert(ResponseAcceptation responseAcceptation) {
     public ResponseAcceptationAssert isEqualTo(CCDResponseAcceptation ccdResponseAcceptation) {
         isNotNull();
 
-        if (!Objects.equals(actual.getFormaliseOption().name(), ccdResponseAcceptation.getFormaliseOption().name())) {
+        if (!Objects.equals(
+            actual.getFormaliseOption().get().name(),
```

Comment:
`Optional.get()` without checking `isPresent()`

Reply:
but this is a test....we can be sure it will be present here no?

Reply:
please use `orElse` orElseThrow`

Reply:
i dont get it lol - we know exactly what behaviour to expect - ill add though anyway